internal/distro/rhel85/distro.go

```diff
@@ -1036,7 +1052,7 @@ func New() distro.Distro {
 	edgeInstallerImgTypeX86_64 := imageType{
 		name:     "edge-installer",
 		filename: "installer.iso",
-		mimeType: "application/x-tar",
+		mimeType: "application/x-iso9660-image",
```

Comment:
Bug fix of previous commit?

Reply:
Yes.  Rebased to keep the history clean.